src/modules/broadcast/broadcast.service.ts

```diff
@@ -153,12 +158,13 @@ export class BroadcastService {
           allCountries = countries.filter((country): country is ICountry => country !== null);
         }
 
-        const session: { allCountries?: ICountry[]; [key: string]: any } =
+        const session: { [key: string]: any } =
           this.commonService.getUserState(Number(userId)) || {};
         session.flow = 'broadcast';
-        session.allCountries = allCountries;
         session.searchType = 'country';
         this.commonService.setUserState(Number(userId), session);
```

Comment:
This also can be moved to cache as now we have it. Not to directly use runcache here, but you can update `setUserState`, `getUserState` etc functions to use runcache.